master/master.cfg

```diff
@@ -347,13 +306,13 @@ def get_halide_install_path(*subpaths):
 
 
 def add_get_halide_source_steps(factory, builder_type):
-    factory.addStep(Git(name='Get Halide source',
-                        locks=[performance_lock.access('counting')],
-                        codebase='halide',
-                        workdir=get_halide_source_path(),
-                        repourl='git://github.com/halide/Halide.git',
-                        branch='master',
-                        mode='incremental'))
+    factory.addStep(GitHub(name='Get Halide source',
```

Comment:
Interesting, is doing this via `GitHub` rather than plain `Git` better/faster/stronger?

Reply:
It's necessary according to the docs if you're using the GitHub change hooks. I didn't question it.